### PR TITLE
Asset scanner routes

### PR DESCRIPTION
**Pull request by xunchuan-liu**

### CODE REVIEW

File: src/asset_tags/routes.rs [hunk 1]
```diff
@@ -45,10 +45,19 @@ async fn update(
     Ok(HttpResponse::Ok().json(asset_tag))
 }
```
Comment (xunchuan-liu) [inline src/asset_tags/routes.rs:47]: Added a delete route for asset_tags

File: README.md [hunk 2]
```diff
@@ -33,11 +33,14 @@ Actix provides the routing and error handling for actions that run through the H
...
 * Your `.env` file needs to define a couple things. The RUST_LOG config is optional depending on what you are looking to test/debug
+    * `AUTH_SECRET` must be defined. Generate a random string at least **48** bytes long for local development.
+    * Do **not** commit sensitive environment variables or tokens.
 ```
-RUST_LOG=asset_api=trace,rest_api=trace,actix=trace,actix_web=debug,diesel_migrations=trace
+RUST_LOG=qsib_asset=trace,info
 DATABASE_URL=postgres://[username]:[password]@localhost:5432/asset_api
 HOST=0.0.0.0
 PORT=6001
+AUTH_SECRET=***************
```
Comment (trueb2) [inline README.md:43]: Why are these changes showing up in the diff? They are already in master.

File: src/main.rs [hunk 3]
```diff
@@ -108,6 +112,8 @@ mod tests {
     async fn test_health_get_without_token() {
         setup();
 
+        log::error!("Bearer {}", ADMIN_USER.token);
+
```
Comment (trueb2) [inline src/main.rs:116]:
```suggestion
```

We don't want to keep this log

**Review by trueb2 — COMMENTED** [review]
Aside from the comments, this looks good. This is what we need for each route to get started.

**Comment by xunchuan-liu** [comment]
Anything else I should change before the merge?

**Review by trueb2 — APPROVED** [review]

**Pull request merged**